src/patternfly/components/TextInputGroup/text-input-group.scss

```diff
@@ -49,6 +49,13 @@
   color: var(--pf-c-text-input-group--Color, inherit);
   background-color: var(--pf-c-text-input-group--BackgroundColor);
 
+  &.pf-m-hidden,
+  .pf-m-hidden,
+  &[hidden],
+  [hidden] {
```

Comment:
Same here and for `.pf-m-hidden` above - can/should these be limited to direct descendents so it doesn't conflict or do something it shouldn't with some random nested child?